app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml

```diff
@@ -7,7 +7,8 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:orientation="vertical"
-    android:padding="8dp">
+    android:padding="8dp"
+    android:background="@color/color_palette_primary_container_background_color">
```

Comment:
I think this is by mistake. Use component color here.

Reply:
Done.